src/c2s/mongoose_c2s_ranch.erl

```diff
@@ -20,7 +20,7 @@
          }).
 
 -type state() :: #state{}.
--type transport() :: ranch_tcp | ranch_ssl | fast_tls.
+-type transport() :: ranch_tcp | ranch_ssl | fast_tls | just_tls.
```

Comment:
Technically `just_tls` is a wrapper we made around OTP's ssl to make it look just like `fast_tls`. But `ranch_ssl` is also a wrapper around OTP's ssl to make it look like a ranch transport. They should be unified, initially I designed it to be made into `ranch_ssl`, so probably `just_tls` just needs some renaming.